tests/cephfs/clients/mds_conf_modifying.py

```diff
@@ -180,6 +181,11 @@ def run(ceph_cluster, **kw):
             )
             if output[0].rstrip() != target[1]:
                 raise CommandFailed(f"Failed to set {client_conf} to {value}")
+
+        rhbuild = config.get("rhbuild")
+        if "7" in rhbuild:
```

Comment:
The config value set to "0" is applicable to both 6.1 and 7.1 series, any reason to have the build check here?

Reply:
Noted